components/skip-link.tsx

```
@@ -0,0 +1,35 @@
+ * 
+ * @param targetId - ID of the element to skip to (defaults to 'main-content')
+ * @param className - Additional classes to apply to the component
+ */
+export function SkipLink({
+  targetId = "main-content",
+  className = "",
+}: {
+  targetId?: string;
+  className?: string;
+}) {
+  const defaultProps = getSkipLinkProps(targetId);
```

ESLint

[error] 'defaultProps' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)